src/components/organisms/GlobalSidebar/hooks.ts

```diff
@@ -0,0 +1,19 @@
+import { useRef, useEffect } from "react";
+
+export const useSidebar = ({ onClose }: { onClose: () => void }) => {
+  const sidebarRef = useRef<HTMLDivElement>(null);
+  const triggerRef = useRef<HTMLButtonElement>(null);
+
+  useEffect(() => {
+    document.addEventListener("click", (e) => {
+      const isChild = sidebarRef.current?.contains(e.target as Node);
+      const isTrigger = triggerRef?.current?.contains(e.target as Node);
```

Comment:
The second one is needed, the first one is not 🙂 
```suggestion
      const isTrigger = triggerRef.current?.contains(e.target as Node);
```

Reply:
@macfrei 
Thanks!

Reply:
@macfrei 
Done!